c2corg_api/views/validation.py

```diff
@@ -333,7 +331,10 @@ def _add_associations(
 
 
 def _is_parent_of_association(main_document_type, other_document_type):
-    if (main_document_type, other_document_type) in valid_associations:
+    # book is always main_d_t=parent?
+    if main_document_type == 'b' and other_document_type in valid_associations:
```

Comment:
Hmm, I am not sure why you would even need a special case for books. If the associations with books are listed in `valid_associations`, it should be detected properly.

But thinking about it: We should test that associations between books and books are handled correctly.